src/model/entities/test/entity-factory.ts

```diff
@@ -0,0 +1,52 @@
+import { Rundown, RundownInterface } from '../rundown'
+import { Segment, SegmentInterface } from '../segment'
+import { Part, PartInterface } from '../part'
+import { Piece, PieceInterface } from '../piece'
+
+export class EntityFactory {
```

Comment:
Is this only meant for testing purporses? (that fact you set ids to `Math.random()` indicates that).

If yes, it would be nice if the name reflected that so a developer won't accidentally use this for produciton code.